operatorcert/pyxis.py

```diff
@@ -46,15 +78,47 @@ def post_with_cert(
     return resp.json()
 
 
-def get_with_cert(url: str, cert_path: str, key_path: str) -> Dict[str, Any]:
-    if not os.path.exists(cert_path):
-        raise Exception(f"Pyxis cert at path {cert_path} is missing.")
-    if not os.path.exists(key_path):
-        raise Exception(f"Pyxis key at path {key_path} is missing.")
+def get(url: str) -> Dict[str, Any]:
```

Comment:
1. Wrong return type
2. You could do 'raise_for_status' also here

Reply:
A comment near the bottom explains why there's no raise_for_status: 
```
# Not raising exception for error statuses, because GET request can be used to check
# if something exists. We don't want a 404 to cause failures.
```

Reply:
Fixed a return type